functions/Set-DbaDbCompatibility.ps1

```diff
@@ -92,7 +94,7 @@ function Set-DbaDbCompatibility {
         [PSCredential]$SqlCredential,
         [string[]]$Database,
         [Microsoft.SqlServer.Management.Smo.CompatibilityLevel]$Compatibility,
-        [ValidateSet(9, 10, 11, 12, 13, 14, 15)]
+        [ValidateSet(9, 10, 11, 12, 13, 14, 15, 16)]
         [int]$TargetCompatibility,
```

Comment:
This parameter should just be removed with 2.0 release coming we can deal with breaking changes in this manner. It then removes the need to maintain a validate set as the strong typing of the `Microsoft.SqlServer.Management.Smo.CompatibilityLevel` class does that for us now.